lib/collins_auth.rb

```diff
@@ -6,7 +6,7 @@
 module Collins
   module Authenticator
     def self.setup_client(options = {:prompt => false})
-      if options[:config_file] and not File.readable? options[:config_file]
+      if options[:config_file] and not File.readable? File.expand_path(options[:config_file])
```

Comment:
doesn't this also need to go where the config is loaded?

Reply:
Thank you for the catch Roy - I've added another commit to address this.

Reply:
As for the documentation, I would be happy to update it where you see fit - just not sure exactly what part you're referencing that needs to be updated.